.env

```diff
@@ -35,4 +35,7 @@ _APP_FUNCTIONS_CPUS=1
 _APP_FUNCTIONS_MEMORY=128
 _APP_FUNCTIONS_MEMORY_SWAP=128
 _APP_MAINTENANCE_INTERVAL=86400
-_APP_USAGE_STATS=enabled
\ No newline at end of file
+_APP_MAINTENANCE_EXECUTION_LOG_RETENTION=1209600
```

Comment:
The Appwrite way is more like: `_APP_MAINTENANCE_RETENTION_EXECUTION`

Reply:
Same for all..

Reply:
Yup

Reply:
b